pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -391,17 +396,46 @@ func manageKubeControllerManagerConfig(ctx context.Context, client corev1client.
 	return resourceapply.ApplyConfigMap(ctx, client, recorder, requiredConfigMap)
 }
 
-func manageClusterPolicyControllerConfig(ctx context.Context, client corev1client.ConfigMapsGetter, recorder events.Recorder, operatorSpec *operatorv1.StaticPodOperatorSpec) (*corev1.ConfigMap, bool, error) {
+func manageClusterPolicyControllerConfig(ctx context.Context, client corev1client.CoreV1Interface, recorder events.Recorder, operatorSpec *operatorv1.StaticPodOperatorSpec) (*corev1.ConfigMap, bool, error) {
 	configMap := resourceread.ReadConfigMapV1OrDie(bindata.MustAsset("assets/kube-controller-manager/cluster-policy-controller-cm.yaml"))
 	defaultConfig := bindata.MustAsset("assets/config/default-cluster-policy-controller-config.yaml")
+	kcmService := resourceread.ReadServiceV1OrDie(bindata.MustAsset("assets/kube-controller-manager/svc.yaml"))
+	configYamls := [][]byte{
+		defaultConfig,
+		operatorSpec.ObservedConfig.Raw,
+	}
+
+	servingCertName := ""
+	if kcmService.Annotations != nil {
+		servingCertName = kcmService.Annotations[ServingCertSecretAnnotation]
+	}
+
+	if len(servingCertName) == 0 {
+		return nil, false, fmt.Errorf("missing %s annotation in %s/%s service", kcmService.Namespace, kcmService.Name, ServingCertSecretAnnotation)
+	}
+
+	_, err := client.Secrets(operatorclient.TargetNamespace).Get(ctx, servingCertName, metav1.GetOptions{})
+
+	if err != nil && !apierrors.IsNotFound(err) {
+		return nil, false, err
+	} else if apierrors.IsNotFound(err) {
+		// Should only apply when starting the cluster so cluster-policy-controller is able to annotate openshift-service-ca namespace.
+		// Then service-ca controller should start and create serving-cert.
+		// We will put the serving-cert into the config as soon as it appears which will then trigger new installer.
+
+		klog.V(1).Info("serving-cert not found: falling back to default self-signed certificate in cluster-policy-controller")
+		configOverride := "{\"servingInfo\": { \"certFile\": \"\", \"keyFile\": \"\"} }"
+		configYamls = append(configYamls, []byte(configOverride))
```

Comment:
to trigger defaulting here: https://github.com/openshift/library-go/blob/master/pkg/controller/controllercmd/cmd.go#L181

Reply:
would be handy to have that link/location to that defaulting in comment.